packages/loading/src/directive.js

```diff
@@ -91,6 +91,7 @@ exports.install = Vue => {
 
     update: function(el, binding) {
       if (binding.oldValue !== binding.value) {
+        el.instance.setText(el.getAttribute('element-loading-text'));
```

Comment:
Thanks for contributing. This line should go outside the `if` block because everything inside this block will not execute until the binding value changes.